### PR TITLE
Fixed: Removed PO reallocation job from UI(#32129kj)

### LINKED ISSUE #282: Remove PO reallocation job
## What is the motivation for adding/enhancing this feature?
The PO reallocation job doesn't actually solve any problem and may cause confusion for the user. So we should remove it from UI.

Found on the pre-order page at the bottom.

### PR DESCRIPTION
**Pull request by shashwatbangar**
### Related Issues

Closes #282 

### Short Description and Why It's Useful

Removed reallocation section from preorder page as it do not solve any problem and can create confusion to the user

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

**Review by k2maan — APPROVED** [review]

**Comment by dt2patel** [comment]
We should also remove data for this job from the env files etc.

**Comment by shashwatbangar** [comment]
> We should also remove data for this job from the env files etc.

Done sir

**Review by dt2patel — APPROVED** [review]
Looks good to me @adityasharma7

File: .env.example [hunk 1]
```diff
@@ -4,7 +4,7 @@ VUE_APP_CACHE_MAX_AGE=3600
 VUE_APP_VIEW_SIZE=10
 VUE_APP_JOB_FREQUENCY_TYPE={"BOPIS_CORRECTION":"default","PRE_ORDER_CTLG":"default","ADD_PRODR_TG_SHPFY":"default","REMV_PRODR_TG_SHPFY":"default","ADD_PRODR_SHPG_DTE_SHPFY":"default","REMV_PRODR_SHPG_DTE_SHPFY":"default","BACK_ORDER_CTLG":"default","ADD_BACKODR_TG_SHPFY":"default","IMP_NEW_ORDERS":"default","IMP_CANCELLED_ORDERS":"default","REMV_BACKODR_TG_SHPFY":"default","ADD_BACKODR_SHPG_DTE_SHPFY":"default","IMP_CANCELLED_ITEMS":"default","REMV_BACKODR_SHPG_DTE_SHPFY":"default","REALLOC_PRODR":"default","AUTO_RELSE_DAILY":"default","IMP_PAYMENT_STATUS":"default","IMP_RETURNS":"default","UPLD_CMPLT_ORDRS":"default","UPLD_CNCLD_ORDRS":"default","UPLD_REFUNDS":"default","NTS_PRMS_DT_CHNG":"default","REJ_ORDR":"default","UNFIL_ORDERS":"default","AUTO_CNCL_DAL":"default","BTCH_BRKR_ORD":"default","LIST_PRE_ORDER":"slow","LIST_BACK_ORDER":"slow","IMP_PRDTS":"default","SYNC_PRDTS":"slow","HARD_SYNC":"slow","IMP_PRDTS_BLK":"default","IMP_ORDERS_BLK":"default"}
```
Comment (adityasharma7) [inline .env.example:5]: We missed removing it from here

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**